tests/test_env.py

```diff
@@ -1,9 +1,32 @@
 import pytest
 
-from game_mechanics import Env
+from poker.texas_holdem import DeltaEnv
```

Comment:
I believe this should be `delta_poker.env_wrapper`?

Guessing this means tests weren't run pre-push?

Perhaps we should make sure tests are run pre-push? Idk how easy this is to do outside of everyone adding stuff to their `.git` dir indepently - can it be added to `pre-commit`?

Reply:
It's in the .pre-commit-config.yaml in the game template. I forgot that you need to run an installation command to init the pre-push hooks. Have updated the template readme with this command